### PR TITLE
[Merged by Bors] - Add Proposer Cache Pruning & POS Activated Banner

### PR DESCRIPTION
**Pull request by ethDreamer**
## Issue Addressed

The proposers cache wasn't being pruned. Also didn't have a celebratory banner for the merge :smile:

## Banner
pos_log_panda

### CODE REVIEW

**Review by michaelsproul — APPROVED** [review]
Looks great!

File: beacon_node/beacon_chain/src/block_verification.rs [hunk 1]
```diff
@@ -81,6 +81,30 @@ use types::{
...
+
+       .o.                   .   o8o                         .                 .o8
+      .888.                .o8   `"'                       .o8                "888
+     .8"888.     .ooooo. .o888oooooo oooo    ooo .oooo.  .o888oo .ooooo.  .oooo888
+    .8' `888.   d88' `"Y8  888  `888  `88.  .8' `P  )88b   888  d88' `88bd88' `888
+   .88ooo8888.  888        888   888   `88..8'   .oP"888   888  888ooo888888   888
+  .8'     `888. 888   .o8  888 . 888    `888'   d8(  888   888 .888    .o888   888
+ o88o     o8888o`Y8bod8P'  "888"o888o    `8'    `Y888""8o  "888"`Y8bod8P'`Y8bod88P"
+
+"#;
```
Comment (lucas-manuel) [inline beacon_node/beacon_chain/src/block_verification.rs:106]: Small proposed change:

```suggestion
  ;"   ^;     ;'   ",                                           ;"   ^;     ;'   ",
  ;    s$$$$$$$s     ;                                          ;    s$$$$$$$s     ;
  ,  ss$$$$$$$$$$s  ,'  ooooooooo.    .oooooo.   .oooooo..o     ,  ss$$$$$$$$$$s  ,'
  ;s$$$$$$$$$$$$$$$     `LFG   `Y88. d8P'  `Y8b d8P'    `Y8     ;s$$$$$$$$$$$$$$$
  $$$$$$$$$$$$$$$$$$     LFG   .d88'LFG      LFGY88bo.          $$$$$$$$$$$$$$$$$$
 $$$$P""Y$$$Y""W$$$$$    LFGooo88P' LFG      LFG `"YLFG8o.     $$$$P""Y$$$Y""W$$$$$
 $$$$  p"LFG"q  $$$$$    LFG        LFG      LFG     `"Y88b    $$$$  p"LFG"q  $$$$$
 $$$$  .$$$$$.  $$$$     LFG        `88b    d88'oo     .d8P    $$$$  .$$$$$.  $$$$
  $$DcaU$$$$$$$$$$      oLFGo        `Y8bood8P' 8""LFG88P'      $$DcaU$$$$$$$$$$
    "Y$$$"*"$$$Y"                                                 "Y$$$"*"$$$Y"
        "$b.$$"                                                       "$b.$$"
       .o.                   .   o8o                         .                 .o8
      .LFG.                .o8   `"'                       .o8                "LFG
     .8"LFG.     .ooooo. .oLFGoooooo oooo    ooo .oooo.  .oLFGoo .ooooo.  .ooooLFG
    .8' `LFG.   d88' `"Y8  LFG  `LFG  `88.  .8' `P  )88b   LFG  d88' `88bd88' `LFG
   .88oooLFG8.  LFG        LFG   LFG   `88..8'   .oP"LFG   LFG  LFGoooLFGLFG   LFG
  .8'     `LFG. LFG   .o8  LFG . LFG    `LFG'   d8(  LFG   LFG .LFG    .oLFG   LFG
 o88o     oLFG8o`Y8bod8P'  "LFG"oLFGo    `8'    `YLFG""8o  "LFG"`Y8bod8P'`Y8bod88P"

"#;
```

Comment (lucas-manuel) [inline beacon_node/beacon_chain/src/block_verification.rs:106]: Small proposal, just between the eyes
```
    ,,,         ,,,                                               ,,,         ,,,
  ;"   ^;     ;'   ",                                           ;"   ^;     ;'   ",
  ;    s$$$$$$$s     ;                                          ;    s$$$$$$$s     ;
  ,  ss$$$$$$$$$$s  ,'  ooooooooo.    .oooooo.   .oooooo..o     ,  ss$$$$$$$$$$s  ,'
  ;s$$$$$$$$$$$$$$$     `888   `Y88. d8P'  `Y8b d8P'    `Y8     ;s$$$$$$$$$$$$$$$
  $$$$$$$$$$$$$$$$$$     888   .d88'888      888Y88bo.          $$$$$$$$$$$$$$$$$$
 $$$$P""Y$$$Y""W$$$$$    888ooo88P' 888      888 `"Y8888o.     $$$$P""Y$$$Y""W$$$$$
 $$$$  p"LFG"q  $$$$$    888        888      888     `"Y88b    $$$$  p"LFG"q  $$$$$
 $$$$  .$$$$$.  $$$$     888        `88b    d88'oo     .d8P    $$$$  .$$$$$.  $$$$
  $$DcaU$$$$$$$$$$      o888o        `Y8bood8P' 8""88888P'      $$DcaU$$$$$$$$$$
    "Y$$$"*"$$$Y"                                                 "Y$$$"*"$$$Y"
        "$b.$$"                                                       "$b.$$"
       .o.                   .   o8o                         .                 .o8
      .888.                .o8   `"'                       .o8                "888
     .8"888.     .ooooo. .o888oooooo oooo    ooo .oooo.  .o888oo .ooooo.  .oooo888
    .8' `888.   d88' `"Y8  888  `888  `88.  .8' `P  )88b   888  d88' `88bd88' `888
   .88ooo8888.  888        888   888   `88..8'   .oP"888   888  888ooo888888   888
  .8'     `888. 888   .o8  888 . 888    `888'   d8(  888   888 .888    .o888   888
 o88o     o8888o`Y8bod8P'  "888"o888o    `8'    `Y888""8o  "888"`Y8bod8P'`Y8bod88P"
```

**Pull request closed without merging**